### PR TITLE
New theme proposal

### PR DESCRIPTION
**Pull request by xakraz**
A more "sober" theme than the current one...

But this one has
* Day/night theme support
* Direct mention to all social medias used by p7t
* Hugo posts about the projects to more info about each of them

Screenshot_2020-06-04 p7t
Screenshot_2020-06-04 p7t(1)
Screenshot_20200604_101329

### CODE REVIEW

**Comment by xakraz** [comment]
@guilhem this theme use the "Extended" version of Hugo.
I don't really know how to specify it in the Netlify config file.

More over, it seems that it needs at least to use the latest image version of their builder
* https://docs.netlify.com/configure-builds/common-configurations/#hugo
* https://rahilw.dev/blog/hugo-extended-on-netlify/

:pray:

File: config-html5up.toml [hunk 1]
```diff
@@ -0,0 +1,218 @@
+# Site settings
```
Comment (guilhem) [inline config-html5up.toml]: what is this config file?

File: config.toml [hunk 2]
```diff
@@ -1,218 +1,136 @@
...
+      disclaimer = ""
+
+  # Social config about the author
+  [params.social]
+    GitHub = "p7t"
+
+    Linkedin = ""
+    Slidershare = ""
+
+    Twitter = ""
```
Comment (guilhem) [inline config.toml]:
```suggestion
    Twitter = "devobs_podcast"
```

File: config.toml [hunk 3]
```diff
@@ -1,218 +1,136 @@
...
+      # whether to show social links
+      social = true
+      # LoveIt NEW | 0.2.0 disclaimer (HTML format is supported)
+      disclaimer = ""
+
+  # Social config about the author
+  [params.social]
+    GitHub = "p7t"
+
+    Linkedin = ""
```
Comment (guilhem) [inline config.toml]:
```suggestion
    Linkedin = "company/18291222"
```

File: config.toml [hunk 4]
```diff
@@ -1,218 +1,136 @@
...
+  # Syntax Highlighting (https://gohugo.io/content-management/syntax-highlighting)
+  [markup.highlight]
+    # false is a necessary configuration (https://github.com/dillonzq/LoveIt/issues/158)
+    noClasses = false
+
+# Author config
+[author]
+  name = "p7t"
+  email = "contact@p7t.tech"
+  link = "https://github.com/p7t"
```
Comment (guilhem) [inline config.toml]: booo no new line :p

File: config-html5up.toml (hunk 1 above)
Comment (xakraz) [reply config-html5up.toml]: Right, it shouldn't be there ^^ Fixing

**Comment by xakraz** [comment]
@guilhem How do we do to see a "preview" of Netlify rendering before merging and breaking things ? :stuck_out_tongue_closed_eyes:

**Comment by guilhem** [comment]
> @guilhem How do we do to see a "preview" of Netlify rendering before merging and breaking things ?

I fix running build, but with error `Error during Hugo extended_0.72.0 install: invalid version vextended_0.72.0: Malformed version: extended_0.72.0` now

**Comment by guilhem** [comment]
As I add netlify app after opening this PR, can you try to do another one?

**Comment by xakraz** [comment]
> > @guilhem How do we do to see a "preview" of Netlify rendering before merging and breaking things ?
> 
> I fix running build, but with error `Error during Hugo extended_0.72.0 install: invalid version vextended_0.72.0: Malformed version: extended_0.72.0` now

I see ... But I don't really know how to specify we want the `extended` version ...

Screenshot_20200604_113222

**Comment by xakraz** [comment]
Reopen for Netlify integration

**Pull request closed without merging**